registration.md

```diff
@@ -253,7 +253,14 @@ This describes how we handle the response, after an account has been
 successfully created.
 
 * If the request is `Accept: application/json`, the response should be status
-  200 with an empty body.
+  200 with a JSON body.  The body should only contain a single JSON field:
+  `status`.  It should look something like this:
+
+```json
+{
+  "status": "VERIFIED"
```

Comment:
I'm fine with this, the need is there.  When we were initially looking at this, we were taking a minimalistic approach to only return information if needed.  Since there is a need here, we can do it!

++